.github/workflows/pythonpackage.yml

```diff
@@ -26,13 +26,8 @@ jobs:
         gdal-config --version
     - name: Install system dependencies (macOS)
       if: runner.os == 'macOS'
-      # GDAL 3.2.1
-      # See https://cmichel.io/how-to-install-an-old-package-version-with-brew/
       run: |
-        brew tap-new $USER/local-gdal
-        brew extract --version=3.2.1 gdal $USER/local-gdal
-        brew install gdal@3.2.1
-        brew install pkg-config proj geos
+        brew install gdal pkg-config proj geos
```

Comment:
So, problem already solved after a few days it seems?

Reply:
Yes ! 

> https://github.com/nextgis/pygdal/pull/61

Reply:
Yes ! 

> Pygal (Add GDAL 3.2.2 # 61)